CITATION.cff

```diff
@@ -0,0 +1,53 @@
+cff-version: 1.2.0
+title: DLA-Future
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+authors:
+  - given-names: Alberto
+    family-names: Invernizzi
+  - given-names: Raffaele
+    family-names: Solcà
+    orcid: 'https://orcid.org/0009-0009-9346-4376'
```

Comment:
Maybe swap?

Reply:
Absolutely: https://github.com/eth-cscs/DLA-Future/pull/1082#issuecomment-1895837863

> I would at least propose that we move @rasolca up to first place. I have no opinion on the order of other authors (can also be alphabetical).

Any thoughts on order of others?

Reply:
Some suggestions that come to my mind:
- project members, active contributors to the release, non-active contributors
- project members in alphabetical order (first name)
- active contributors sorted w.r.t. number of lines / alphabetically
- non-active contributors in reverse order w.r.t. their last contribution

This would currently lead for the next version to:
RS | AI, AR, JB, MS, RM | GG | AF, HS, TN, LQ, AR, AC

Reply:
Sounds good to me. Perhaps worth a quick discussion in the next meeting.

Reply:
I updated the above comment as discussed yesterday and fixed the list.

Reply:
I've updated the list to the above order.